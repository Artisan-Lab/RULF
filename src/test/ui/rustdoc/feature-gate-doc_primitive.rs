// check-pass
#[doc(primitive = "usize")]
//~^ WARNING `doc(primitive)` should never have been stable
//~| WARNING hard error in a future release
/// Some docs
mod usize {}

fn main() {}
