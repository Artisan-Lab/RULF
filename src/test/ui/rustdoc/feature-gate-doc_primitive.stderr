warning: `doc(primitive)` should never have been stable
  --> $DIR/feature-gate-doc_primitive.rs:2:7
   |
LL | #[doc(primitive = "usize")]
   |       ^^^^^^^^^^^^^^^^^^^
   |
   = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
   = note: for more information, see issue #82730 <https://github.com/rust-lang/rust/issues/82730>
   = note: `#[warn(invalid_doc_attributes)]` on by default

warning: 1 warning emitted

